package.json

```diff
@@ -15,9 +14,11 @@
     "@graasp/apps-query-client": "3.4.15",
     "@graasp/sdk": "4.9.0",
     "@graasp/ui": "4.17.1",
-    "@mui/icons-material": "5.16.7",
-    "@mui/lab": "5.0.0-alpha.173",
-    "@mui/material": "5.16.7",
+    "@mui/icons-material": "6.1.4",
+    "@mui/lab": "6.0.0-beta.12",
+    "@mui/material": "6.1.4",
```

Comment:
Is the move to mui 6 intentional ?

Reply:
It was to test because vscode was unhappy, but finally I only had to restart it, but I forgot to undo my changes, so thanks!